packages/nimble-components/src/table/index.ts

```diff
@@ -84,7 +90,36 @@ export class Table<
         this.setTableData(newData);
     }
 
-    public idFieldNameChanged(
+    public override connectedCallback(): void {
+        super.connectedCallback();
+        this.virtualizer.connectedCallback();
+        this.observeColumns();
+        this.validateColumnIds();
+    }
+
+    public override disconnectedCallback(): void {
```

Comment:
I'm pretty sure there is a `super.disconnectedCallback()` to call in the base class (at least other FoundationElement based components are doing so).

[Doesn't look like there is a TS feature to help ensure we call super](https://github.com/microsoft/TypeScript/issues/21388) but may be a [lint rule](https://github.com/typescript-eslint/typescript-eslint/issues/684) soon. Best we have is double-checking overrides I guess (even though it was missed when it first went in lol).